.github/workflows/build-package.yml

```
@@ -89,6 +89,12 @@ env:
   NO_DEFAULT_FEATURES: ${{ inputs.NO_DEFAULT_FEATURES }}
   BUILD_DIR: ${{ github.workspace }}/built.d
 
+run-name: >-
+  Build Package
+  ${{ inputs.crate }}
+  ${{ inputs.version }} 
+  (${{ inputs.build_os }})
```

I created this from my phone, so I went with a mulit-line string that yml parsers concatenate into a single line. It should still have the same result as proposed in #415.

Just in case this conflicts with style guidelines:
```suggestion
run-name: Build Package ${{ inputs.crate }} ${{ inputs.version }} (${{ inputs.build_os }})
```
Both are equivalent per YAML specifications.